draft-ietf-rats-eat.md

```diff
@@ -1174,6 +1174,8 @@ See {{common-types}} for OID and URI encoding.
 EAT's may be used in the context of several different applications.  The "intuse"
 claim provides an indication to an EAT consumer about  the intended usage
 of the token. This claim can be used as a way for an application using EAT to internally distinguish between different ways it uses EAT.
+5 possible values for "intuse" are currently defined indicated 5 different intended use indications, but additional values above 5 can
+be defined and registered pending IANA-administered expert review (see {{iana-cons}}).
```

Comment:
EAT's should be EATs in the first sentence. I'd change the wording on the new language slightly:

```
Five possible values for "intuse" are currently defined, providing 5 different intended use indications. However, additional values can be defined and registered pending IANA-administered expert review (see {{iana-cons}}).
```

I don't think just referencing "expert review" answers the question regarding a registry, so I'd also add something about a possible future registry.

```
At present, there is not a registry for "intuse" values. If a future Internet Draft defines any additional values, a registry should be created and initialized with the five values defined in this document plus any new values.
```

Reply:
Please review updates

Reply:
Works for me. Thanks.